endoreg_db/models/media/video/video_file.py

```
@@ -430,24 +456,30 @@ def delete(self, using = ..., keep_parents = ...):
-        # Call the superclass delete method to ensure proper deletion
-        # This will handle the database deletion and any additional cleanup defined in the parent class
-        # Using super() to ensure we call the correct method in the parent class hierarchy
-        from django.db.models.deletion import Collector
-        collector = Collector(using=using)
-        collector.collect([self], keep_parents=keep_parents)
-        collector.delete()
-        return super().delete(using, keep_parents)
+        # Use proper database connection
+        if using is None:
+            using = 'default'
+        lock_path = Path(video.get_raw_file_path()).with_suffix(video.get_raw_file_path().suffix + ".lock")
```

Pylint (4.0.0)

[error] Undefined variable 'video'

(E0602)